packages/postgres-database/src/simcore_postgres_database/models/products_prices.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+
+from ._common import NUMERIC_KWARGS, column_created_datetime
+from .base import metadata
+from .products import products
+
+#
+# - Every product has an authorized price
+# - The price is valid from the creation date until a new price is created
+# - No rows are deleted!
+# - If a product has no price, it is assumed zero
+#
+
+products_prices = sa.Table(
+    "products_prices",
+    metadata,
+    sa.Column(
+        "product_name",
+        sa.String,
+        sa.ForeignKey(
+            products.c.name,
+            name="fk_products_prices_product_name",
+            ondelete="RESTRICT",
+            onupdate="CASCADE",
+        ),
+        nullable=False,
+        doc="Product name",
+    ),
+    sa.Column(
+        "usd_per_credit",
+        sa.Numeric(**NUMERIC_KWARGS),  # type: ignore
+        nullable=False,
+        doc="Price in USD/credit >=0",
+    ),
+    sa.Column(
+        "authorized_by",
+        sa.String,
+        nullable=False,
+        doc="For the moment a comment on the product owner (PO) who authorized this price",
+    ),
+    column_created_datetime(timezone=True),
+    sa.CheckConstraint(
+        "usd_per_credit >= 0", name="non_negative_usd_per_credit_constraint"
+    ),
```

Comment:
Idea: I would also add some text "comment" column where we can store some custom human readable info for example why we are changing the price (for example because of Christmas discounts or something similar)

Reply:
that is the `authorized_by` column for.

Reply:
renamed